libswirl/hw/pvr/Renderer_if.cpp

```diff
@@ -205,12 +205,15 @@ static bool rend_frame(u8* vram, TA_context* ctx) {
         rend_init_renderer(vram);
     }
 
-    bool proc = renderer->Process(ctx);
-
-	if (!proc || !ctx->rend.isRTT) {
-		// If rendering to texture, continue locking until the frame is rendered
-		pend_rend = false;
-		re.Set();
+    bool proc = true;

```

Comment:
These are here because slightly different timing / automagic frameskip between LLE and HLE